src/System.Dynamic.Runtime/src/System.Dynamic.Runtime.csproj

```diff
@@ -119,8 +113,22 @@
     <Compile Include="System\Runtime\CompilerServices\RuleCache.cs" />
     <Compile Include="System\Runtime\CompilerServices\RuntimeOps.cs" />
   </ItemGroup>
-  <ItemGroup>
+  <ItemGroup Condition=" '$(IsInterpreting)' != 'true' ">
+    <Compile Include="$(CommonPath)\System\Dynamic\Utils\CollectionExtensions.Map.cs">
+      <Link>Common\System\Dynamic\Utils\CollectionExtensions.Map.cs</Link>
+    </Compile>
+    <Compile Include="$(CommonPath)\System\Linq\Expressions\Compiler\AssemblyGen.cs">
+      <Link>Common\System\Linq\Expressions\Compiler\AssemblyGen.cs</Link>
+    </Compile>
+
     <None Include="project.json" />
   </ItemGroup>
+  <ItemGroup Condition=" '$(IsInterpreting)' == 'true' ">
+    <Compile Include="$(CommonPath)\System\Dynamic\Utils\DelegateHelpers.cs">
+      <Link>Common\System\Dynamic\Utils\DelegateHelpers.cs</Link>
+    </Compile>
+
+    <None Include="projectInterpreting.json" />
```

Comment:
Yeah, we discussed offline.  Today if you want to have a custom project.json you can set the ProjectJson and ProjectLockJson properties before importing buildtools and they'll be honored by [restore/resolve targets](https://github.com/dotnet/buildtools/blob/9b7720013790b5b42f52c7e896f8ad2921e35a1a/src/Microsoft.DotNet.Build.Tasks/Targets/packageresolve.targets#L26)